edg_acoustics/time_integration.py

```diff
@@ -39,16 +40,69 @@ def step_dt(self):
         pass
 
 class TSI_TI(TimeIntegrator):
-    def __init__(self, AcousticSimulation_object, CFL: float=CFL_Default):
+    def __init__(self, AcousticSimulation_object, L_operator: typing.Callable[[numpy.array, numpy.array, numpy.array, numpy.array]], Nt: int, CFL: float=CFL_Default):
```

Comment:
You want to remove the sim object here, right? I think we can do that. 
I didn't know we could pass in argument likes this before, thanks!

Reply:
Yes. Magical right?